tests/python/tir-transform/test_tir_transform_merge_dynamic_shared_memory_allocations.py

```diff
@@ -456,7 +456,7 @@ def func(
 class TestSimpleAllocNoReuse(tvm.testing.CompareBeforeAfter):
     """Test alloc and free within the same scope."""
 
-    transform = tvm.tir.transform.MergeDynamicSharedMemoryAllocations()
```

Comment:
I am not sure the change here is intended `MergeDynamicSharedMemoryAllocations`

Reply:
I was getting an error that there is no such pass